README.md

```diff
@@ -33,7 +33,7 @@ shotgun $sel $*
 
 ## shotgun vs maim
 
-- Only PNG is supported
+- Only PNG and PAM are supported
```

Comment:
Would be nice to document the "why PAM" part in the readme.

Reply:
Let's go patch the image crate to encode bmp faster

Reply:
Why don't we support more by the way? Shouldn't it be as easy as just handling more cases and relying on the `image` crate?

Reply:
The idea is that shotgun should support the smallest amount of features possible and then leave more complex tasks to imagemagick or any other program.

Reply:
If we already support choosing between two extensions I don't know why disallow choosing between N of them.

Reply:
Because more image encoders make the binary bigger. PNG is kept for backwards compatibility while PAM is for performance.

Reply:
Ah, Rust is smart enough to not include in the binary functions that it doesn't use from external crates. That's fair then.

Reply:
We explicitly choose which ones to enable in Cargo.toml.

Reply:
I wish there was a better way to find out what features a crate has.

Reply:
As far as I know, reading its Cargo.toml is your best bet. Maybe it's listed on crates.io?

Reply:
Yes, [reading their `Cargo.toml`](https://github.com/PistonDevelopers/image/blob/0f5b967bc5bc63c1a51d09e2fcebc833c5ef6b89/Cargo.toml#L65-L80) is your best bet currently. I wonder if an issue for [crates.io](https://crates.io) about this could be worthy.

Reply:
Ah, there's already an issue for this at https://github.com/rust-lang/crates.io/issues/465.